Makefile

```diff
@@ -40,18 +44,22 @@ deps:
 	go get -u github.com/goadesign/gorma
 
 generate: $(DESIGNS)
-	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
-	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
-	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
-	go-bindata-assetfs -debug assets/...
-	godep get
+	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
```

Comment:
Understood! Great idea.

Konrad Kleine | Software Engineer
Red Hat GmbH
phone: +4921195439689
email: kkleine@redhat.com

On Thu, Jun 30, 2016 at 9:28 AM, Thomas Mäder <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/almighty/almighty-core/pull/4#discussion_r69083897>:
>
> > @@ -40,18 +44,22 @@ deps:
> >  	go get -u github.com/goadesign/gorma
> >
> >  generate: $(DESIGNS)
> > -	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
> > -	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
> > -	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
> > -	go-bindata-assetfs -debug assets/...
> > -	godep get
> > +	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
>
> It's ok if we assume that there is one global GOPATH. I don't agree that
> this should be the case. We might have multiple workspaces with go code.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/almighty/almighty-core/pull/4/files/6809e69838cf74a34dc19ee246ed4f57b66d4168#r69083897>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALzgBszQEEvvsLq5y7zNYQTAhO5rSilks5qQ3AhgaJpZM4I8524>
> .
>